sources/postgres/infoschema.go

```diff
@@ -463,9 +463,9 @@ func toType(dataType string, elementDataType sql.NullString, charLen sql.NullInt
 		// TODO: handle case of multiple array bounds.
 	case charLen.Valid:
 		return schema.Type{Name: dataType, Mods: []int64{charLen.Int64}}
-	case dataType == "numeric" && numericPrecision.Valid && numericScale.Valid && numericScale.Int64 != 0:
+	case numericPrecision.Valid && numericScale.Valid && numericScale.Int64 != 0:
```

Comment:
Is the datatype check not relevant here anymore ?

Reply:
Its not relevant because we need to store the column length for all the datatypes in the source database